Automattic/wp-calypso
# Site Profiler: Update styles for Results Header section

Related to https://github.com/Automattic/dotcom-forge/issues/7280
Resolves https://github.com/Automattic/dotcom-forge/issues/7384
Resolves https://github.com/Automattic/dotcom-forge/issues/7294
Resolves https://github.com/Automattic/dotcom-forge/issues/7383

## Proposed Changes

* Create the Results Header component with correct styles
* Connect the buttons for `open the report` and `check another site`
* Add one of the backgrounds 

|Desktop | Mobile|
|-------|------|
| ![CleanShot 2024-05-22 at 18 57 19@2x](https://github.com/Automattic/wp-calypso/assets/3519124/aefd7999-f604-4196-ac67-0f5b66f761b3)  |  ![CleanShot 2024-05-22 at 19 00 47@2x](https://github.com/Automattic/wp-calypso/assets/3519124/9481fef5-6906-481c-a313-7b79fe3d18f3) |

## Why are these changes being made?


* To create the new results header section as per design

## Testing Instructions



* Apply this patch and in your local environment navigate to `/site-profiler`
* Add a URL and click on `Check site`
* Confirm the styles of the results header match Figma in [Desktop](https://www.figma.com/design/P7Jjk4nbUTahUk8aHkG6NQ/Performance-Tool?node-id=712-25699&m=dev) and Mobile
* Click on the `Access full site report - It’s free`
* Check the report form is opened at the bottom
* Close the form
* Click on the `Check another site` link
* Check the page navigates to the landing page

#### Things to be done as part of other tasks:
- Add the correct icon next to the domain https://github.com/Automattic/dotcom-forge/issues/7281
- Change the background image based on score https://github.com/Automattic/dotcom-forge/issues/7298

## Pre-merge Checklist
